### ISSUE TITLE
Views counter double each refresh

### ISSUE THREAD

**Issue by max246** [DevAlien/dripr-ui#26]
Looking at this link: https://dripr.io/file/rJyIa_omg
Every time you refresh the page, the counter add 2 visits each time.

**Comment by DevAlien** [comment]
True,  this I due to a reload client side which should not happen

**Comment by ianaz** [comment]
Yes pretty annoying
